docs/dropout.md

```diff
@@ -1 +1,361 @@
-Coming soon.
+## Fitting a noisy sinusoid with quantum dropout
+
+Here we will demonstrate an implemention [quantum dropout](https://arxiv.org/abs/2310.04120), for the case of fitting a noisy sine function.
+
+```python exec="on" source="material-block" html="1"
+
+from __future__ import annotations
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from matplotlib import ticker
+
+# from sklearn.preprocessing import MinMaxScaler
+from torch import manual_seed, optim, tensor
+
+import pyqtorch as pyq
+from pyqtorch.circuit import DropoutQuantumCircuit
+from pyqtorch.parametric import Parametric
+
+manual_seed(12345)
+np.random.seed(12345)
+
+
+class MinMaxScaler:
```

Comment:
Ok, for all of the  functions and classes here I would suggest to add docstrings and comments (simpler functions may have just a sentence of what they are), but this way their functionalities will be clearer to the user.

Reply:
Also, for this documentation treat it as a proper tutorial, which implies that you need to add more context on each of the steps taken. Check the tutorials in qadence.